js/ui/discoveryFeedButton.js

```diff
@@ -51,12 +51,10 @@ const DiscoveryFeedButton = new Lang.Class({
     Extends: St.Button,
 
     _init: function() {
-        let iconFile = Gio.File.new_for_uri('resource:///org/gnome/shell/theme/discovery-feed-open-tab.png');
-        let gicon = new Gio.FileIcon({ file: iconFile });
-        this._icon = new St.Icon({ gicon: gicon,
-                                   style_class: 'discovery-feed-icon',
-                                   track_hover: true });
+        this._icon = new St.Icon({style_class: 'discovery-feed-tile-icon',
```

Comment:
If you set the background with CSS, you should not need this `St.Icon` child on the button anymore. Try simply setting the background image on the `.discovery-feed-tile` style class of the button (which however I would rename to `.discovery-feed-button`)